docs/app/src/search.js

```diff
@@ -23,43 +25,43 @@ angular.module('search', [])
         angular.forEach(hits, function(hit) {
           var area = hit.area;
 
-          var limit = (area == 'api') ? 40 : 14;
+          var limit = (area === 'api') ? 40 : 14;
           results[area] = results[area] || [];
-          if(results[area].length < limit) {
+          if (results[area].length < limit) {
             results[area].push(hit);
           }
         });
 
         var totalAreas = 0;
-        for(var i in results) {
+        // eslint-disable-next-line no-unused-vars
+        for (var i in results) {
```

Comment:
>
> Wtf? Why not var totalAreas = results.length?
>
It's an object, not an array. I changed that locally to
`Object.keys(results).length`.